### PR TITLE
Updating pycortex installation

### PR DESCRIPTION
**Pull request by gunjanbaid**
Based on updated instructions from @eickenberg.

### CODE REVIEW

**Comment by eickenberg** [comment]
Thanks!

File: user-image/connectors/neuro.bash [hunk 1]
```diff
@@ -14,6 +14,7 @@ pip install --no-cache-dir cython==0.26
 git clone https://github.com/eickenberg/pycortex.git
 cd pycortex
 # commit on data8 branch
-git checkout 4fa348c1309d3c90ed220c6811f0fee617ea83a9
+git checkout f4e5fd4f94570c1195fcf4616c7783740e34adb6
 python setup.py install
 cd .. && rm -rf pycortex
+rm -rf ~/.config/pycortex
```
Comment (eickenberg) [inline user-image/connectors/neuro.bash]: For a completely fresh install this folder might not exist. Is this OK? Are the scripts set to run through anyway?

Comment (gunjanbaid) [reply user-image/connectors/neuro.bash]: The `-f` will suppress errors if the folder doesn't exist.

Comment (eickenberg) [reply user-image/connectors/neuro.bash]: oh cool, ok!

Comment (gunjanbaid) [reply user-image/connectors/neuro.bash]: @eickenberg Could you test out the updates on staging.datahub.berkeley.edu? If it looks good, I can merge into datahub.

**Comment by eickenberg** [comment]
I checked on staging.

- the changes that were implemented in this PR have taken effect (path to pycortex store is correct)
- possibly the port forwarding mechanism for having the 3D viewer does not work on staging. I get
```
Internal server error: [Errno 111] Connection refused
```
This error may be nonexistent outside staging.

Bottom line: the changes of the given PR have taken effect. So I think this can be merged.

CC @samyag1

**Comment by gunjanbaid** [comment]
@yuvipanda @ryanlovett Could you confirm that it is ok to merge into prod, despite the above error?

**Comment by eickenberg** [comment]
Maybe useful for debug:
The link that doesn't work: http://datahub.berkeley.edu/user/michael.eickenberg/proxy/58097/mixer.html
The notebook from which it stems: http://staging.datahub.berkeley.edu/user/michael.eickenberg/notebooks/Untitled.ipynb

**Comment by eickenberg** [comment]
actually, I think I see what is happening. Pycortex just generates a datahub link by default and there is not pycortex active there

**Comment by eickenberg** [comment]
Yep: http://staging.datahub.berkeley.edu/user/michael.eickenberg/proxy/58097/mixer.html works!

**Comment by gunjanbaid** [comment]
Sounds good, merging now.

**Pull request merged**